src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -85,6 +87,24 @@ public Command parseCommand(String userInput) throws ParseException {
         case RemarkCommand.COMMAND_WORD:
             return new RemarkCommandParser().parse(arguments);
 
+        case ListingAddCommand.COMMAND_WORD_PREFIX:
```

Comment:
I am not sure if this big chunk of code can be put into the list command parser. I understand speed was your priority here.